client/src/js/components/bhCashboxSelect.js

```diff
@@ -36,7 +36,7 @@ function CashboxSelectController(Cashbox, Notify) {
     $ctrl.label = $ctrl.label || 'FORM.SELECT.CASHBOX';
 
     // load all Cashbox
-    Cashbox.read(null, { detailed: 1 })
+    Cashbox.read(null, { detailed: 1, user_id : Session.user.id })
```

Comment:
Hmm... the user could in theory alter this, since it's on the client.  I wonder what we should do about that.  We could use `req.session.user.id` on the server, but that would limit ability to download all cashboxes.

We need a better security model in general.  I do not have specific proposals for this route, so this is a non-blocking comment.